<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalTemplate</class>
 <widget class="QWidget" name="CalTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Template for Calendar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="previewGroupBox">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>417</height>
      </size>
     </property>
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="Digikam::CalWidget" name="calendarWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="settingsGroupBox">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="paperSizeLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Paper Size:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="paperSizeCombo">
            <item>
             <property name="text">
              <string>A4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>US Letter</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="resolutionLabel">
            <property name="text">
             <string>Resolution:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="resolutionCombo">
            <item>
             <property name="text">
              <string>High</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Low</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="imagePosButtonGroup">
          <property name="title">
           <string>Image Position</string>
          </property>
          <property name="current" stdset="0">
           <number>-1</number>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="topRadio">
               <property name="text">
                <string>&amp;Top</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="leftRadio">
               <property name="text">
                <string>&amp;Left</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rightRadio">
               <property name="text">
                <string>&amp;Right</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="drawLinesCheckBox">
          <property name="text">
           <string>Draw lines on calendar</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ratioLayout">
          <item>
           <widget class="QLabel" name="ratioLabel">
            <property name="text">
             <string>Image to text ratio:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="ratioSlider">
            <property name="minimum">
             <number>50</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="fontLayout">
          <item>
           <widget class="QLabel" name="fontLabel">
            <property name="text">
             <string>Font:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFontComboBox" name="fontCombo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="yearLabel">
            <property name="text">
             <string>Year:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="yearSpin"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="monthBox">
       <property name="title">
        <string>Select Images</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="monthBoxLayout"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Left click on a month to select an image, right click to clear an image.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Digikam::CalWidget</class>
   <extends>QWidget</extends>
   <header>calwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>paperSizeCombo</tabstop>
  <tabstop>resolutionCombo</tabstop>
  <tabstop>topRadio</tabstop>
  <tabstop>leftRadio</tabstop>
  <tabstop>rightRadio</tabstop>
  <tabstop>drawLinesCheckBox</tabstop>
  <tabstop>ratioSlider</tabstop>
  <tabstop>fontCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
